perllib/Open311/PostServiceRequestUpdates.pm

```diff
@@ -41,18 +39,25 @@ sub fetch_bodies {
 }
 
 sub construct_open311 {
-    my ($self, $body) = @_;
-    my $o = Open311->new($self->open311_params($body));
-    $self->current_open311($o);
+    my ($self, $body, $comment) = @_;
+    my $o = Open311->new($self->open311_params($body, $comment));
+    return $o;
 }
 
 sub open311_params {
-    my ($self, $body) = @_;
+    my ($self, $body, $comment) = @_;
+
+    my $conf = $body;
+    if ($comment) {
```

Comment:
When would this ever not have a comment?

Reply:
Ah, well, umm, the test suite in t/open311/post-service-request-updates.t calls this fn directly to check its cobrand output for various bodies, and it didn't seem worth passing in a fake comment even though the code always does. I guess I could factor out second half of this function to a different one and use that but didn't seem worth it.

Reply:
yup, makes sense.